.buildkite/scripts/common.sh

```diff
@@ -60,10 +53,6 @@ xpack_heartbeat_changeset=(
   "^x-pack/heartbeat/.*"
   )
 
```

Comment:
remove the above lines, we shouldn't be reintroducing xpack/heartbeat (see the file on 8.14 as it is now https://github.com/elastic/beats/blob/3918ab802c2aac432451b241d0e50d76809d4554/.buildkite/scripts/common.sh#L26)